falco.yaml

```diff
@@ -139,6 +139,22 @@ rules_file:
   - /etc/falco/falco_rules.local.yaml
   - /etc/falco/rules.d
 
+# [Experimental] `rule_matching`
```

Comment:
Maybe we should find a proper place for this new config, don't know if here is where we want it

Reply:
I would suggest moving it in one of the sections below, perhaps around/before the rate limiter options.

Also, I would avoid leaking knowledge about the evttype bucket indexing here in this comment. I think that for end users what matters is just understanding that Falco stops or not at the first rule match. The rest is an internal implementation detail on which we need to honor this interface.

Reply:
Addressed, let me know what you think now